oak_runtime/src/lib.rs

```diff
@@ -435,6 +435,7 @@ pub struct Runtime {
     aux_servers: Mutex<Vec<AuxServer>>,
 
     /// Queue of introspection events in chronological order.
+    #[allow(dead_code)]
```

Comment:
I wish this could have been avoided, but I cannot think of a better alternative.